mobile-app/app/screens/AppNavigator/screens/Loans/screens/PaybackLoanScreen.tsx

```diff
@@ -622,145 +457,196 @@ function TransactionDetailsSection ({
+          light={tailwind('text-gray-500')}
+          iconType='MaterialIcons'
+          name={!isExpanded ? 'expand-more' : 'expand-less'}
+          size={24}
+          testID='toggle_resulting_col'
+        />
+      </ThemedView>
+      {isExpanded &&
+        <ThemedView
+          style={tailwind('px-4 mx-4 py-1 border-b')}
+          dark={tailwind('bg-gray-800 border-gray-700')}
+          light={tailwind('bg-white border-gray-200')}
+        >
+          <TextRow
+            containerStyle={{
+              style: tailwind('flex flex-row pb-1'),
+              dark: tailwind('bg-gray-800 border-gray-700'),
+              light: tailwind('bg-white border-gray-200')
+            }}
+            lhs={{
+              value: translate('screens/PaybackLoanScreen', 'Vault ID'),
+              themedProps: rowStyle.lhsThemedProps,
+              testID: 'lhs_vault_id'
+            }}
             rhs={{
-              value: amountToPayInLoanToken.minus(outstandingBalance).toFixed(8),
-              testID: 'text_excess_amount',
-              suffixType: 'text',
-              suffix: displaySymbol
+              value: vault.vaultId,
+              testID: 'text_vault_id',
+              numberOfLines: 1,
+              ellipsizeMode: 'middle',
+              themedProps: rowStyle.rhsThemedProps
             }}
+            textStyle={tailwind('text-xs font-normal')}
           />
-        )}
-      <NumberRow
-        lhs={translate('screens/PaybackLoanScreen', 'Resulting {{displaySymbol}} Balance', { displaySymbol: selectedPaymentToken.tokenDisplaySymbol })}
-        rhs={{
-          value: BigNumber.max(resultingBalance, 0).toFixed(8),
-          testID: 'text_resulting_balance',
-          suffixType: 'text',
-          suffix: selectedPaymentToken.tokenDisplaySymbol
-        }}
-      />
-      <TextRow
-        lhs={translate('screens/PaybackLoanScreen', 'Vault ID')}
-        rhs={{
-          value: vault.vaultId,
-          testID: 'text_vault_id',
-          numberOfLines: 1,
-          ellipsizeMode: 'middle'
-        }}
-        textStyle={tailwind('text-sm font-normal')}
-      />
-      <NumberRow
-        lhs={translate('screens/PaybackLoanScreen', 'Remaining loan amount')}
-        rhs={{
-          value: BigNumber.max(outstandingBalance.minus(amountToPayInLoanToken), 0).toFixed(8),
-          testID: 'text_resulting_loan_amount',
-          suffixType: 'text',
-          suffix: displaySymbol
-        }}
-      />
-      {resultingColRatio.isLessThan(0)
-        ? (
           <TextRow
-            lhs={translate('screens/PaybackLoanScreen', 'Resulting collateralization')}
+            containerStyle={{
+              style: tailwind('flex flex-row pb-1'),
+              dark: tailwind('bg-gray-800 border-gray-700'),
+              light: tailwind('bg-white border-gray-200')
+            }}
+            lhs={{
+              value: translate('screens/PaybackLoanScreen', 'Min. Col. Ratio'),
```

Comment:
Can we use sentence case, to be consistent with other label
```suggestion
              value: translate('screens/PaybackLoanScreen', 'Min. col. ratio'),
```

Reply:
Updated